### ISSUE TITLE
Fake News List Correction

### ISSUE THREAD

**Issue by chadgauth** [StevenBlack/hosts#540]
www.washingtonexaminer.com shows up in the fake news list. From some research it is listed as high factual reporting and right center bias, but does not appear to have fake news.

**Comment by funilrys** [comment]
Hello @chadgauth and welcome here!

As the fake new extensions come from `https://github.com/marktron/fakenews` ([cf](https://github.com/StevenBlack/hosts/blob/dad43783a03bb262dc88283f815dd3b7be4ed216/extensions/fakenews/update.json)).

I invite you to post your issue to https://github.com/marktron/fakenews/issues where @marktron will take the time to take a look and review this request.

Have a nice day/night.

_____________
# Occurences of `www.washingtonexaminer.com`
```
./extensions/fakenews/hosts <== Entry point
./alternates/fakenews-porn/hosts
./alternates/fakenews-gambling/hosts
./alternates/fakenews-social/hosts
./alternates/fakenews-gambling-porn-social/hosts
./alternates/fakenews-gambling-porn/hosts
./alternates/fakenews/hosts
./alternates/fakenews-porn-social/hosts
./alternates/fakenews-gambling-social/hosts
```

**Comment by funilrys** [comment]
Also, Steven @StevenBlack is that list really accurate as the last update date is `29 Nov 2017` ([cf](https://github.com/marktron/fakenews/commits/master))? (just saw that ...)

**Comment by ScriptTiger** [comment]
@marktron, bump ^

**Comment by StevenBlack** [comment]
Mark @marktron must be spring skiing.  It's been a brutal winter here in Central Canada, I'll give him a couple more days  😎

**Comment by StevenBlack** [comment]
This is fixed in [Release 1.3.12](https://github.com/StevenBlack/hosts/releases/tag/v1.3.12).

Closing.
